WordPress/Classes/Extensions/UITextField+TextHelper.swift

```diff
@@ -0,0 +1,11 @@
+import UIKit
+
+extension UITextField
+{
+    func nonNilTrimmedText() -> String {
+        guard let str = text?.trim() else {
+            return ""
+        }
+        return str
+    }
```

Comment:
I can see that, tho I think I prefer to avoid optionals where ever possible.  Its a nice helper method to reduce the amount of optional unwrapping that would otherwise be required.